10067 - Radiant vanguard

bread6464 opened the pull request:
Still working on it, just did all the normal attack dialogue and beg dialogue for now

bread6464 committed:
10067 - All normal attacks and beg dialogue

bread6464 committed:
All masturbation dialogues

bread6464 committed:
Radiant vanguard - 1HP Temptation scene 1

bread6464 committed:
radiant vanguard - line 120

bread6464 committed:
Comma

Theguysayhi commented:
Just thought I'd pop in for this ^^

How's this PR doing @bread6464? Do you need any additional assistance with it?

bread6464 commented:
Hey @Theguysayhi I've been busy IRL I couldn't translate more. I'll try to do what I can. If this is blocking the release of the next translation patch and you want someone else to work on it, just tell me and you can take it. I pushed all the translations I already did, that's about 30% of the work (and that should cover all combat related dialogue). I think what's left is the "after defeat" stuff.

I didn't realize the file was like 400 lines long when I took it, I should've probably started with something a bit smaller and more manageable.

HotHeadEmperor committed:
Update 10067.txt

Upload to transfer work to Wiz. Thanks man!

HotHeadEmperor committed:
Update 10067.txt

WisDZ committed:
Update 10067.txt

Adding changes - some lines are shakier than others & I haven't tested any of the lines yet. Line 274, 282, and 291 especially need another pass, but I'll add notes after all of the lines have been looked at.

WisDZ committed:
Ellipsis fix

WisDZ committed:
Fix - line 277

WisDZ committed:
Update 10067.txt

WisDZ committed:
Update 10067.txt

WisDZ commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+やっほーお兄さん♪その様子だとあの後ちゃんと出さなかったんだね❤\nそんな状態で立ってたら次のダメージで確実にイっちゃうでしょ❤,Yoohoo; Mister Master~♪ From the looks of it; you're still all pent up from before; aren't you~❤ If you keep standing there in that state; you're definitely going to end up cumming from the next hit; you know that; right?❤
+どうする？このまま次のターンまで待ってわたしにトドメさしてもらう？\nでもマスターがわたしに攻撃の命令をするとは限らないし・・・,So what are you gonna do? Wait until the next turn for me to finish you off? But there's no guarantee my Master will even order me to attack...
+そうだ！お兄さんの方から攻撃してよ❤\nそしたらカウンターでお兄さんの望みどおりと・ど・め❤さしてあげる♪,Oh; I know! How about you attack me instead; mister!❤ Then I'll counter and fi~nish... you... off~❤ Just the way you've been wanting.♪
+あ、ああ❤・・・ちゃん❤先導者ちゃんっ❤,Ah; ahh...❤ ...-chan.❤ Vanguard-chan~❤
```
Not sure how to localize this one!

Theguysayhi replied:
Hmm what about something like this?
It might be better to keep the vibes and drop the name entirely.

```suggestion
あ、ああ❤・・・ちゃん❤先導者ちゃんっ❤,Ah; ahh...❤ I... P-Please...❤ I can't~❤
```

WisDZ commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+それに自分から負けに来てくれるなんてとってもお利口さん\nわたしの思い通りに動いてくれるなんてもうお兄さんはわたしの味方同然だね❤,And coming to lose to me on your own like that... What a smart little thing you are~ Moving exactly as I want you to... Mister; you're practically on my side already~❤
+そんなえらい子にはご褒美をあげないと❤\n味方にしかしてあげないわたしの特別な応援\nお兄さんのためだけにしてあげるよ❤いくよー♪,A good boy like you needs to be rewarded~❤ My special cheer that I only give to allies... I'll do it just for you; mister~❤ Here we go~♪
+がんばれー❤がんばれー❤おちんちんもっと硬くなれー❤\n足で踏まれてもへっちゃらの丈夫なおちんちんになーれっ❤\nふれっ❤ふれっ❤ふれ～～～～❤,Do your best!❤ Do your best!❤ I wanna feel that cock get even harder~❤ Hard enough that you won't even flinch when I grind my feet all over it~❤ Ready~❤ Set~❤ Heeeere we go~~~❤
+はーい完成❤あはは❤おちんちんの守備力上がっちゃった❤\nガチガチになって生半可な攻撃が通らない光属性おちんちんになれたねー❤,Aaand done~❤ Ahaha!❤ Your cock's defense is boosted now~❤ Now you're rock hard; and with a light-attribute dick that can withstand any half-hearted attack~❤
```
This line needs a second-look

Theguysayhi replied:
Hm what about this? 

```suggestion
はーい完成❤あはは❤おちんちんの守備力上がっちゃった❤\nガチガチになって生半可な攻撃が通らない光属性おちんちんになれたねー❤,Aaand done~❤ Ahaha!❤ Your cock's defense has been increased. With how rock hard you are now; your new light-attribute dick will be able to shrug off any half-hearted attack~❤
```

WisDZ commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+声を聴くたびに股間がぎゅんぎゅんってなっちゃうね❤\n防御力があがるたびにおちんちんの硬さも増してなんだかすごく興奮しちゃう❤,Every time you hear my voice your crotch throbs wildly~❤ And each time your defense goes up; your cock keeps getting harder and harder; too... It's getting me so worked up!❤
+でも強くなったはずのおちんちんさん、すっごく苦しそう。\n防御力があがったせいで刺激が弱く感じちゃうんだね❤早くイキたいよーってさっきから先端がピクピクしてる❤,But your cock... Even though it should be stronger... It looks like it's really suffering right now. Because your defense went up; all that stimulation ends up feeling too weak; doesn't it~❤ Your tip's been desperately twitching and quivering this whole time; practically crying out "I wanna cum already~!"❤
+弱い刺激だけじゃイキづらいよね。だから今度は精子さんの応援してあげる❤そしたらお兄さんすぐにイっちゃうと思うけど別にいいよね、もうがまんできないみたいだし❤それじゃあ・・・いくよ・・・,Weak stimulation alone won't make you cum easily; right? So this time; I'll cheer for your sperm on for you~❤ You'll probably end up cumming right away when I do; but that's totally fine; right? You can't hold back anymore anyway~❤ So then... here I go...
+ふれーふれー精子さん！がんばっておちんちんから昇ってきてぇ❤\nびゅびゅっと勢いよく飛び出てわたしの靴を孕ませちゃえー❤\nがんばれがんばれ、わーーー❤❤❤,Go; go sperm! Work hard climbing up from that cock~❤ Spray out vigorously and impregnate my shoes~❤ Go for it; go for it; waaaa~❤❤❤
```
I struggled with a lot of the cheers - got stuck on this one and moved on, but it likely still needs work

Giratena replied:
How about this? ^^

```suggestion
ふれーふれー精子さん！がんばっておちんちんから昇ってきてぇ❤\nびゅびゅっと勢いよく飛び出てわたしの靴を孕ませちゃえー❤\nがんばれがんばれ、わーーー❤❤❤,Go-go sperm; climb up high! Work your way up from that guy's cock with pride!❤ Shoot out strong; make a splash! Cover my shoes in your sticky white stash!❤ You can do it; yes you can! Go-go-goooo!❤❤❤
```

WisDZ commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+楽しいコトたくさん考えよーね❤\nわたしにおちんちん踏まれちゃったときのこととか\n応援されておちんちんカチカチにされちゃったこととか❤,Let's think about lots of fun things; okay~❤ Like when your cock ended up getting trampled under my feet... Or when my cheering ended up getting you all stiff and rock hard~❤
+そうやってどんどん幸せになっていけばいいんだよ❤\nもうデュエルのこととか妹さんのことは忘れてお兄さんが望むことをこれからしていこうね❤,Just like that; just keep getting happier and happier... That's all you need to do!❤ Forget about dueling and your little sister... From now on; let's just do whatever you want~❤
+こうして僕は強さを求めることをやめ、いつしか快感のためだけにデュエルをする人間に成り下がっていった。,I stopped seeking strength altogether. Before long; I had become someone who only dueled for the sake of pleasure.
+弱い己を肯定し続けた故に最後には最弱のマスターとして人生の幕を閉じるのであった。,Because I continued to indulge in my weakness; I eventually closed the final chapter of my life as the weakest Master there ever was.
```
There's probably a better way to word this line

Giratena replied:
Hm, what about this?

```suggestion
弱い己を肯定し続けた故に最後には最弱のマスターとして人生の幕を閉じるのであった。,While indulging in my short-comings; the last chapter of my life came to a close; and I ended up becoming one of the weakest cardmasters out there.
```

WisDZ reviewed (commented):
The last of the lines now have a translation, but we'll definitely need to do an overall pass to get a consistent tone, since we had multiple folks translating different chunks of the character. (The localization of お兄さん is especially inconsistent at the moment)

I also haven't play-tested yet!

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+[2].このカードが自分フィールド上に存在する限り、自分の光属性モンスターが受ける戦闘ダメージは常に200ポイントダウンする。,While this card is on your field; your Light attribute monsters receive 200 less damage when engaged in battle. 
+[3].１ターンに１度、発動できる。自分フィールド上のすべての光属性モンスターの攻撃力を200ポイントアップする。,Once per turn; increase the attack power of all the Light attribute monsters on your field by 200.
+あたしは味方を応援するのが得意なの。\n当然、敵のあなたは応援してあげないわ。\nでもね、代わりに、ここを握って応援してあげる❤,I'm very good at encouraging my allies. Obviously; you are my enemy so I can't cheer you on. However; I can instead encourage you by grabbing you down there~❤
+だっておちんちんって、わたしたちの味方みたいなものでしょ？\n強い男の人をしばりつけて、女の子に勝てなくしちゃう、\nたのもしい味方❤,I mean; your penis is just like an ally right? A very dependable ally that restrains strong men; rendering them unable to win against girls!❤
```
```suggestion
だっておちんちんって、わたしたちの味方みたいなものでしょ？\n強い男の人をしばりつけて、女の子に勝てなくしちゃう、\nたのもしい味方❤,After all; your penis is essentially on my side right? A very dependable ally that restrains strong men; rendering them unable to win against girls!❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+あたしは味方を応援するのが得意なの。\n当然、敵のあなたは応援してあげないわ。\nでもね、代わりに、ここを握って応援してあげる❤,I'm very good at encouraging my allies. Obviously; you are my enemy so I can't cheer you on. However; I can instead encourage you by grabbing you down there~❤
+だっておちんちんって、わたしたちの味方みたいなものでしょ？\n強い男の人をしばりつけて、女の子に勝てなくしちゃう、\nたのもしい味方❤,I mean; your penis is just like an ally right? A very dependable ally that restrains strong men; rendering them unable to win against girls!❤
+ほらぁ、おちんちんさん頑張れ～、しこしこ～❤\nあなたのご主人様をもっともっと弱くしてぇ、\nどぴゅーってエナジー出しちゃおうねー❤,Theeere; do your best Mister Penis~; stroke; stroke~❤ Make your master weaker and weaker; and pour out your energy for us ok?❤
+この旗ねー、折りたたんでいつも持っているから、\nわたしのおっぱいの匂いがついているんだよ。\nとってもあまぁい、女の子の匂いだよ❤嗅いでみたいでしょ？,See this flag? I fold it and always keep it with me; which means the scent of my breasts permeates it. A very sweeeet girly scent.❤ You want to sniff it; right?
```
```suggestion
この旗ねー、折りたたんでいつも持っているから、\nわたしのおっぱいの匂いがついているんだよ。\nとってもあまぁい、女の子の匂いだよ❤嗅いでみたいでしょ？,See this flag? I fold it and always keep it with me; which means the scent of my breasts has rubbed off on it. A very sweeeet girly scent.❤ You want to sniff it; right?
```

Giratena replied:
```suggestion
この旗ねー、折りたたんでいつも持っているから、\nわたしのおっぱいの匂いがついているんだよ。\nとってもあまぁい、女の子の匂いだよ❤嗅いでみたいでしょ？,See this flag? I keep it folded up and close to my chest; which means it has a similar smell to my breasts. A very sweeeet girly scent.❤ You want to sniff it; right?
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+だっておちんちんって、わたしたちの味方みたいなものでしょ？\n強い男の人をしばりつけて、女の子に勝てなくしちゃう、\nたのもしい味方❤,I mean; your penis is just like an ally right? A very dependable ally that restrains strong men; rendering them unable to win against girls!❤
+ほらぁ、おちんちんさん頑張れ～、しこしこ～❤\nあなたのご主人様をもっともっと弱くしてぇ、\nどぴゅーってエナジー出しちゃおうねー❤,Theeere; do your best Mister Penis~; stroke; stroke~❤ Make your master weaker and weaker; and pour out your energy for us ok?❤
+この旗ねー、折りたたんでいつも持っているから、\nわたしのおっぱいの匂いがついているんだよ。\nとってもあまぁい、女の子の匂いだよ❤嗅いでみたいでしょ？,See this flag? I fold it and always keep it with me; which means the scent of my breasts permeates it. A very sweeeet girly scent.❤ You want to sniff it; right?
+ならほら、旗であなたのこと包んじゃう❤\nこうすれば動けないしー、一方的に攻め放題♪\nわたしって頭いいー♪,Here; I'll wrap you up in it~❤ If I do this; you won't be able to move at all; a very one-sided offence.♪ I'm so smart~♪
```
```suggestion
ならほら、旗であなたのこと包んじゃう❤\nこうすれば動けないしー、一方的に攻め放題♪\nわたしって頭いいー♪,Here; I'll wrap you up in it~❤ If I do this; you won't be able to move at all; a very one-sided offense.♪ I'm so smart~♪
```

Giratena replied:
```suggestion
ならほら、旗であなたのこと包んじゃう❤\nこうすれば動けないしー、一方的に攻め放題♪\nわたしって頭いいー♪,Here; I'll wrap you up in it~❤ If I do this; you won't be able to move or fight back at all~♪ I'm so smart~♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+この旗ねー、折りたたんでいつも持っているから、\nわたしのおっぱいの匂いがついているんだよ。\nとってもあまぁい、女の子の匂いだよ❤嗅いでみたいでしょ？,See this flag? I fold it and always keep it with me; which means the scent of my breasts permeates it. A very sweeeet girly scent.❤ You want to sniff it; right?
+ならほら、旗であなたのこと包んじゃう❤\nこうすれば動けないしー、一方的に攻め放題♪\nわたしって頭いいー♪,Here; I'll wrap you up in it~❤ If I do this; you won't be able to move at all; a very one-sided offence.♪ I'm so smart~♪
+ふぐ、はうぅう・・・❤,Aammmh; haauu...❤
+くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,Eh eh; your penis that got bigger is pushing the flag up.❤ So; you got aroused by my scent~❤ Here; I'll stroke you through the flag; stroke; stroke~❤ stroke; stroke~❤ I'll stroke you plenty and make you feel good~❤
```
```suggestion
くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,*Giggle*; your penis is pushing the flag up with its erection.❤ So; you got aroused by my scent~❤ Here; I'll stroke you through the flag; stroke; stroke~❤ stroke; stroke~❤ I'll stroke you plenty and make you feel good~❤
```

Giratena replied:
```suggestion
くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,*Giggle*; your erect penis is making my flag go up!❤ So; you're getting aroused by my scent after all~❤ Here; I'll stroke you through my flag; stroke; stroke~❤ Stroke; stroke~❤ I'll stroke it plenty and make you feel good~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ならほら、旗であなたのこと包んじゃう❤\nこうすれば動けないしー、一方的に攻め放題♪\nわたしって頭いいー♪,Here; I'll wrap you up in it~❤ If I do this; you won't be able to move at all; a very one-sided offence.♪ I'm so smart~♪
+ふぐ、はうぅう・・・❤,Aammmh; haauu...❤
+くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,Eh eh; your penis that got bigger is pushing the flag up.❤ So; you got aroused by my scent~❤ Here; I'll stroke you through the flag; stroke; stroke~❤ stroke; stroke~❤ I'll stroke you plenty and make you feel good~❤
+えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Ha! Stomping attack~❤ I'll keep pressing on you from above; feel good from that~❤
```
```suggestion
えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Ha! Stomping attack~❤ Enjoy the feeling of me continuously pressing on you from above~❤
```

Giratena replied:
```suggestion
えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Take this! I'll stomp on you~❤ Enjoy the feeling of me pressing my foot on your penis!~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ふぐ、はうぅう・・・❤,Aammmh; haauu...❤
+くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,Eh eh; your penis that got bigger is pushing the flag up.❤ So; you got aroused by my scent~❤ Here; I'll stroke you through the flag; stroke; stroke~❤ stroke; stroke~❤ I'll stroke you plenty and make you feel good~❤
+えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Ha! Stomping attack~❤ I'll keep pressing on you from above; feel good from that~❤
+地面にあるゴミをすりつぶすみたいに、\nぐりぐり♪ぐりぐり♪\nこんなに乱暴にされて感じちゃうんだ❤お兄さん、マゾなんだね～❤,Just like pulverizing trash on the floor; I'll grind; grind!♪ Stomp; stomp!♪ Feeling good from such a violent assault.❤ Big brother; you are such a masochist~❤
```
```suggestion
地面にあるゴミをすりつぶすみたいに、\nぐりぐり♪ぐりぐり♪\nこんなに乱暴にされて感じちゃうんだ❤お兄さん、マゾなんだね～❤,It's just like pulverizing trash on the floor. Grind; grind!♪ Stomp; stomp!♪ Feeling good from such a violent assault...❤ Mister; you are such a masochist; aren't you~❤
```

Giratena replied:
This is good Shrug!

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+くすくす❤おっきくなったおちんちんが旗を押し上げてる❤\nいい匂いで興奮しちゃったんだー❤ほら、旗ごしに\nシコシコ❤シコシコ❤たくさん触って気持ちよくしたげるねー❤,Eh eh; your penis that got bigger is pushing the flag up.❤ So; you got aroused by my scent~❤ Here; I'll stroke you through the flag; stroke; stroke~❤ stroke; stroke~❤ I'll stroke you plenty and make you feel good~❤
+えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Ha! Stomping attack~❤ I'll keep pressing on you from above; feel good from that~❤
+地面にあるゴミをすりつぶすみたいに、\nぐりぐり♪ぐりぐり♪\nこんなに乱暴にされて感じちゃうんだ❤お兄さん、マゾなんだね～❤,Just like pulverizing trash on the floor; I'll grind; grind!♪ Stomp; stomp!♪ Feeling good from such a violent assault.❤ Big brother; you are such a masochist~❤
+靴を脱いで、靴下で踏んであげる。\n靴下のさわりごこち、おちんちんで感じると、\nきもちいいんだよー❤,I'll remove my boots and step on you with my socks. The comforting feeling of my socks on your penis feels good right~❤
```
```suggestion
靴を脱いで、靴下で踏んであげる。\n靴下のさわりごこち、おちんちんで感じると、\nきもちいいんだよー❤,I'll remove my boots and step on you with my socks. The comforting feeling of my socks on your penis feels good; right~?❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+えい、踏みつけ攻撃～❤\n上からぐりぐり～ってされて気持ちよくなっちゃえ❤,Ha! Stomping attack~❤ I'll keep pressing on you from above; feel good from that~❤
+地面にあるゴミをすりつぶすみたいに、\nぐりぐり♪ぐりぐり♪\nこんなに乱暴にされて感じちゃうんだ❤お兄さん、マゾなんだね～❤,Just like pulverizing trash on the floor; I'll grind; grind!♪ Stomp; stomp!♪ Feeling good from such a violent assault.❤ Big brother; you are such a masochist~❤
+靴を脱いで、靴下で踏んであげる。\n靴下のさわりごこち、おちんちんで感じると、\nきもちいいんだよー❤,I'll remove my boots and step on you with my socks. The comforting feeling of my socks on your penis feels good right~❤
+寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,For my big brother that's laying on the floor; I'll touch your crotch with my foot and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you in a way that says "Mister semen; please come~❤ out~❤"
```
```suggestion
寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,For Mister; who's just laying on the floor; I'll touch your crotch with my foot and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you and coax your semen to~ come~❤ out~❤
```

Giratena replied:
```suggestion
寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,Since you're just laying on the floor; I'll press my foot on your crotch and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you and coax your semen to~ come~❤ out~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+地面にあるゴミをすりつぶすみたいに、\nぐりぐり♪ぐりぐり♪\nこんなに乱暴にされて感じちゃうんだ❤お兄さん、マゾなんだね～❤,Just like pulverizing trash on the floor; I'll grind; grind!♪ Stomp; stomp!♪ Feeling good from such a violent assault.❤ Big brother; you are such a masochist~❤
+靴を脱いで、靴下で踏んであげる。\n靴下のさわりごこち、おちんちんで感じると、\nきもちいいんだよー❤,I'll remove my boots and step on you with my socks. The comforting feeling of my socks on your penis feels good right~❤
+寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,For my big brother that's laying on the floor; I'll touch your crotch with my foot and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you in a way that says "Mister semen; please come~❤ out~❤"
+何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many times will it take for you to cum I wonder~❤ Let's see. Here; oooooooone~♪ twooooooo~♪ *Giggle*
```
```suggestion
何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many strokes will it take for you to cum; I wonder~❤ Let's see. Here's oooooooone~♪ twooooooo~♪ *Giggle*
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+靴を脱いで、靴下で踏んであげる。\n靴下のさわりごこち、おちんちんで感じると、\nきもちいいんだよー❤,I'll remove my boots and step on you with my socks. The comforting feeling of my socks on your penis feels good right~❤
+寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,For my big brother that's laying on the floor; I'll touch your crotch with my foot and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you in a way that says "Mister semen; please come~❤ out~❤"
+何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many times will it take for you to cum I wonder~❤ Let's see. Here; oooooooone~♪ twooooooo~♪ *Giggle*
+まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone around here have enormous breasts; so I get a bit overshadowed; but... See; I'm surprisingly big myself; right? If I try; I can sandwich your penis; here I come; hey!❤
```
```suggestion
まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone here has enormous breasts; so I may not compare to them; but... See; I'm surprisingly big myself; right? If I try; I can sandwich your penis between them; here I come; hey!❤
```

Giratena replied:
```suggestion
まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone here has enormous breasts; so I know I can't compare with them... But you have to admit; my boobs are still pretty big; right? If I try; I can sandwich your penis between them; here I come! Hah!❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,For my big brother that's laying on the floor; I'll touch your crotch with my foot and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you in a way that says "Mister semen; please come~❤ out~❤"
+何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many times will it take for you to cum I wonder~❤ Let's see. Here; oooooooone~♪ twooooooo~♪ *Giggle*
+まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone around here have enormous breasts; so I get a bit overshadowed; but... See; I'm surprisingly big myself; right? If I try; I can sandwich your penis; here I come; hey!❤
+すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right.❤ I'll rub you plenty; so don't go away~❤ I'll take one breast; and the other; and... rub; rub!❤ again and again!❤
```
```suggestion
すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right.❤ I'll rub you plenty; so don't try to pull away~❤ I'll take one breast; and the other; and... rub; rub!❤ again and again!❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many times will it take for you to cum I wonder~❤ Let's see. Here; oooooooone~♪ twooooooo~♪ *Giggle*
+まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone around here have enormous breasts; so I get a bit overshadowed; but... See; I'm surprisingly big myself; right? If I try; I can sandwich your penis; here I come; hey!❤
+すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right.❤ I'll rub you plenty; so don't go away~❤ I'll take one breast; and the other; and... rub; rub!❤ again and again!❤
+あはは❤わたしに「気持ちよくしてー」っておねがいしちゃうんだ❤\nお兄さん、かわいいところあるんだね。\n素直になれてえらいよー？,Ahaha~❤ Asking me to make you feel good...❤ Big brother; you have quite a cute side; good job for becoming honest with yourself~
```
```suggestion
あはは❤わたしに「気持ちよくしてー」っておねがいしちゃうんだ❤\nお兄さん、かわいいところあるんだね。\n素直になれてえらいよー？,Ahaha~❤ Asking me to make you feel good...❤ Mister; you have quite a cute side; don't you? Congratulations for being honest with yourself~
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone around here have enormous breasts; so I get a bit overshadowed; but... See; I'm surprisingly big myself; right? If I try; I can sandwich your penis; here I come; hey!❤
+すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right.❤ I'll rub you plenty; so don't go away~❤ I'll take one breast; and the other; and... rub; rub!❤ again and again!❤
+あはは❤わたしに「気持ちよくしてー」っておねがいしちゃうんだ❤\nお兄さん、かわいいところあるんだね。\n素直になれてえらいよー？,Ahaha~❤ Asking me to make you feel good...❤ Big brother; you have quite a cute side; good job for becoming honest with yourself~
+ほら、なにしてほしいのかなー。\nちゃんと大きな声で言ってみてよ。ほらほら♪,Hey; what do you want me to do? Say it with a loud voice; come on; come on!♪
```
```suggestion
ほら、なにしてほしいのかなー。\nちゃんと大きな声で言ってみてよ。ほらほら♪,sO; what do you want me to do? Declare it loudly and proudly! Come on; then.♪
```

Giratena replied:
```suggestion
ほら、なにしてほしいのかなー。\nちゃんと大きな声で言ってみてよ。ほらほら♪,So; what do you want me to do? Declare it loudly and proudly! Come on; then.♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
-ほら、がまんなんてしなくていいんだよー❤\nおにいさんはシコシコがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,
-くすくす❤上手なしこしこだったね❤\nわたしの応援で元気になれた？\nバトルに負けちゃったのに、気持ちよさそうな顔ー❤,
-あは、びゅるるうって射精しちゃったね❤\nどうどう？わたしのおっぱいも悪くなかったでしょ？,
+ふーん、足で踏んでほしいんだー。\nお兄さん、へんたいなんだねー❤,Eeeeh; so you want me to step on you. Big brother is a pervert~❤
```
```suggestion
ふーん、足で踏んでほしいんだー。\nお兄さん、へんたいなんだねー❤,Eeeeh; so you want me to step on you. You're such a pervert; Mister~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
-くすくす❤上手なしこしこだったね❤\nわたしの応援で元気になれた？\nバトルに負けちゃったのに、気持ちよさそうな顔ー❤,
-あは、びゅるるうって射精しちゃったね❤\nどうどう？わたしのおっぱいも悪くなかったでしょ？,
+ふーん、足で踏んでほしいんだー。\nお兄さん、へんたいなんだねー❤,Eeeeh; so you want me to step on you. Big brother is a pervert~❤
+しょうがないからやってあげる❤\nぐりぐり、ごしごしって激しくしちゃうから、\n途中で音をあげないでね♪,Well; if that's what you want; I'll give it to you.❤ I'll roughly grind and stroke you plenty; so don't suddenly start making loud noises ok?
```
```suggestion
しょうがないからやってあげる❤\nぐりぐり、ごしごしって激しくしちゃうから、\n途中で音をあげないでね♪,Well; if that's what you want; I'll give it to you.❤ I'll roughly grind and give you plenty of strokes; so try not to make any sudden noises; okay?♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
-あは、びゅるるうって射精しちゃったね❤\nどうどう？わたしのおっぱいも悪くなかったでしょ？,
+ふーん、足で踏んでほしいんだー。\nお兄さん、へんたいなんだねー❤,Eeeeh; so you want me to step on you. Big brother is a pervert~❤
+しょうがないからやってあげる❤\nぐりぐり、ごしごしって激しくしちゃうから、\n途中で音をあげないでね♪,Well; if that's what you want; I'll give it to you.❤ I'll roughly grind and stroke you plenty; so don't suddenly start making loud noises ok?
+わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. I'll make my big brother's growing penis feel good with my tiny little hands~♪
```
```suggestion
わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. I'll make Mister's enlarged penis feel good between my small little hands~♪
```

Giratena replied:
```suggestion
わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. Your penis is so big and hard my hands can barely fit around it! I'll rub it and make it feel good~♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ふーん、足で踏んでほしいんだー。\nお兄さん、へんたいなんだねー❤,Eeeeh; so you want me to step on you. Big brother is a pervert~❤
+しょうがないからやってあげる❤\nぐりぐり、ごしごしって激しくしちゃうから、\n途中で音をあげないでね♪,Well; if that's what you want; I'll give it to you.❤ I'll roughly grind and stroke you plenty; so don't suddenly start making loud noises ok?
+わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. I'll make my big brother's growing penis feel good with my tiny little hands~♪
+わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze you with my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you are deliberately going out of your way to beg me. You have unusual tastes big brother~❤
```
```suggestion
わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze you with my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you went out of your way to specifically ask me. You have unusual tastes; don't you Mister~?❤
```

Giratena replied:
```suggestion
わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze your dick between my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you went out of your way to specifically ask me. You have unusual tastes; don't you Mister~?❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+しょうがないからやってあげる❤\nぐりぐり、ごしごしって激しくしちゃうから、\n途中で音をあげないでね♪,Well; if that's what you want; I'll give it to you.❤ I'll roughly grind and stroke you plenty; so don't suddenly start making loud noises ok?
+わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. I'll make my big brother's growing penis feel good with my tiny little hands~♪
+わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze you with my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you are deliberately going out of your way to beg me. You have unusual tastes big brother~❤
+いいよ❤かわいいわたしの、ぷにぷにおっぱいで、\nお兄さんのことすぐイかせてあげるんだから❤,But that's fine!❤ My cute self and my squishy breasts will make big brother cum instantly!❤
```
```suggestion
いいよ❤かわいいわたしの、ぷにぷにおっぱいで、\nお兄さんのことすぐイかせてあげるんだから❤,Okay then~❤ With my squishy breasts and cute looks; I'll make you cum in no time; Mister.❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+わたしの手でシコシコしてほしいの？\nふふ、いいわよ。わたしの小っちゃい手で\nお兄さんの大きくなったおちんちん、気持ちよくしたげる♪,You want me to stroke you with my hand? *Giggle*; that's fine with me. I'll make my big brother's growing penis feel good with my tiny little hands~♪
+わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze you with my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you are deliberately going out of your way to beg me. You have unusual tastes big brother~❤
+いいよ❤かわいいわたしの、ぷにぷにおっぱいで、\nお兄さんのことすぐイかせてあげるんだから❤,But that's fine!❤ My cute self and my squishy breasts will make big brother cum instantly!❤
+あん❤おっぱいの中でびゅくびゅく出してる♪\nこんなにたくさん出したら、\nわたしのおっぱいだと余裕ではみ出しちゃうね❤,Ha!❤ You're pouring so much inside my breasts~♪ If you cum that much; it will easily gush out from my breasts~❤
```
あん is a moan, not a laugh

Giratena replied:
Very good spot Shrug ^^

```suggestion
あん❤おっぱいの中でびゅくびゅく出してる♪\nこんなにたくさん出したら、\nわたしのおっぱいだと余裕ではみ出しちゃうね❤,Ahh~❤ You're pouring so much into my breasts~♪ If you cum that much; it'll all spill over~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+わたしのおっぱいで挟んでほしいの？\nふふ、もっとおっぱい大きい子もいるのに、\nわざわざ、わたしに頼むなんて、お兄さん変わってるね❤,You want me to squeeze you with my breasts? *Giggle*; even though there are girls with bigger breasts than mine; you are deliberately going out of your way to beg me. You have unusual tastes big brother~❤
+いいよ❤かわいいわたしの、ぷにぷにおっぱいで、\nお兄さんのことすぐイかせてあげるんだから❤,But that's fine!❤ My cute self and my squishy breasts will make big brother cum instantly!❤
+あん❤おっぱいの中でびゅくびゅく出してる♪\nこんなにたくさん出したら、\nわたしのおっぱいだと余裕ではみ出しちゃうね❤,Ha!❤ You're pouring so much inside my breasts~♪ If you cum that much; it will easily gush out from my breasts~❤
+大きさはそこそこだけど、わたしのパイズリ気持ちよかったでしょ❤\nふふん、戦うまでもなくたっぷりエナジーもらっちゃった❤\nラッキー♪,I'm not that big; but my paizuri felt really good; wouldn't you agree?❤ Eheh; I got so much energy without even having to fight...❤ Lucky me~♪
```
```suggestion
大きさはそこそこだけど、わたしのパイズリ気持ちよかったでしょ❤\nふふん、戦うまでもなくたっぷりエナジーもらっちゃった❤\nラッキー♪,My breasts aren't that big; but my paizuri still felt really good; didn't it?❤ Ehehe; I got so much energy without even having to fight...❤ Lucky me~♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+あん❤おっぱいの中でびゅくびゅく出してる♪\nこんなにたくさん出したら、\nわたしのおっぱいだと余裕ではみ出しちゃうね❤,Ha!❤ You're pouring so much inside my breasts~♪ If you cum that much; it will easily gush out from my breasts~❤
+大きさはそこそこだけど、わたしのパイズリ気持ちよかったでしょ❤\nふふん、戦うまでもなくたっぷりエナジーもらっちゃった❤\nラッキー♪,I'm not that big; but my paizuri felt really good; wouldn't you agree?❤ Eheh; I got so much energy without even having to fight...❤ Lucky me~♪
+あは❤びくんびくんって脈打ってる♪\n足に踏まれて我慢できなかったねー❤,Aha!❤ It's pulsating! You couldn't hold back at all when I stepped on you right~❤
+おちんちんを踏んづけられて射精するなんて、\nお兄さんのへんたーい❤,Cumming from being stepped on... Big brother; you're a pervert~❤
```
```suggestion
おちんちんを踏んづけられて射精するなんて、\nお兄さんのへんたーい❤,Cumming from being stepped on... You're such a pervert; aren't you Mister~?❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+大きさはそこそこだけど、わたしのパイズリ気持ちよかったでしょ❤\nふふん、戦うまでもなくたっぷりエナジーもらっちゃった❤\nラッキー♪,I'm not that big; but my paizuri felt really good; wouldn't you agree?❤ Eheh; I got so much energy without even having to fight...❤ Lucky me~♪
+あは❤びくんびくんって脈打ってる♪\n足に踏まれて我慢できなかったねー❤,Aha!❤ It's pulsating! You couldn't hold back at all when I stepped on you right~❤
+おちんちんを踏んづけられて射精するなんて、\nお兄さんのへんたーい❤,Cumming from being stepped on... Big brother; you're a pervert~❤
+くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's coming out of your penis; there is so much inside my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
```
```suggestion
くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's all gushing out of your penis; there's so much pooling in my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
```

Giratena replied:
```suggestion
くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's all gushing out of your penis and pooling in my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+あは❤びくんびくんって脈打ってる♪\n足に踏まれて我慢できなかったねー❤,Aha!❤ It's pulsating! You couldn't hold back at all when I stepped on you right~❤
+おちんちんを踏んづけられて射精するなんて、\nお兄さんのへんたーい❤,Cumming from being stepped on... Big brother; you're a pervert~❤
+くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's coming out of your penis; there is so much inside my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
+わたしたちのために頑張ってくれてえらいえらい♪\nありがとうねー、なでなで～❤\nあは、またびゅって出した❤素直でえらいねー❤,Good job for doing your best for us Mister Penis.♪ Thank you; I'll pat your head; there; there~❤ aha; it came out again!❤ You're so honest; good job~❤
```
```suggestion
わたしたちのために頑張ってくれてえらいえらい♪\nありがとうねー、なでなで～❤\nあは、またびゅって出した❤素直でえらいねー❤,Good job on doing your best for us; Mister Penis.♪ I'll pat your head as thanks; there; there~❤ Aha; it came out again!❤ You really are so honest; good job~❤
```

Giratena replied:
```suggestion
わたしたちのために頑張ってくれてえらいえらい♪\nありがとうねー、なでなで～❤\nあは、またびゅって出した❤素直でえらいねー❤,Good job on doing your best for us; Mister Penis.♪ I'll pat your head as thanks; there; there~❤ Aha; you just squirted again!❤ What an honest penis you are; good job~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+おちんちんを踏んづけられて射精するなんて、\nお兄さんのへんたーい❤,Cumming from being stepped on... Big brother; you're a pervert~❤
+くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's coming out of your penis; there is so much inside my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
+わたしたちのために頑張ってくれてえらいえらい♪\nありがとうねー、なでなで～❤\nあは、またびゅって出した❤素直でえらいねー❤,Good job for doing your best for us Mister Penis.♪ Thank you; I'll pat your head; there; there~❤ aha; it came out again!❤ You're so honest; good job~❤
+くすくす❤旗にくるまれながら射精しちゃった♪\nおちんちんこんな風にシコシコされてもイっちゃうなんて、\nお兄さんのおちんちん、ずいぶん弱いのね♪,*Giggle*❤ You came while being wrapped up in my flag.♪ Cumming from that kind of stroking; big brother's penis is pretty weak~♪
```
```suggestion
くすくす❤旗にくるまれながら射精しちゃった♪\nおちんちんこんな風にシコシコされてもイっちゃうなんて、\nお兄さんのおちんちん、ずいぶん弱いのね♪,*Giggle*❤ You came while being wrapped up in my flag.♪ Cumming from that kind of stroking; Mister has a pretty weak penis; huh?♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+くすくす❤おちんちんからどぴゅどぴゅ～って\nわたしの手の中にたくさん出ちゃってる❤\nおちんちんさん、応援してもらえてうれしくなっちゃったのかなー❤,*Giggle*; it's coming out of your penis; there is so much inside my hand~❤ Mister Penis; did you get all happy from being cheered on?❤
+わたしたちのために頑張ってくれてえらいえらい♪\nありがとうねー、なでなで～❤\nあは、またびゅって出した❤素直でえらいねー❤,Good job for doing your best for us Mister Penis.♪ Thank you; I'll pat your head; there; there~❤ aha; it came out again!❤ You're so honest; good job~❤
+くすくす❤旗にくるまれながら射精しちゃった♪\nおちんちんこんな風にシコシコされてもイっちゃうなんて、\nお兄さんのおちんちん、ずいぶん弱いのね♪,*Giggle*❤ You came while being wrapped up in my flag.♪ Cumming from that kind of stroking; big brother's penis is pretty weak~♪
+あは、旗から出してあげたら、すっごく蕩けた顔してる❤\nわたしの匂いにつつまれながらしこしこ～ってしてもらえて、\nよかったわね。雑魚モンスターさん❤,Aha; I'll pull you out from the flag; you are making such a melted face.❤ It was nice to be stroked while being wrapped in my scent right? You small fry monster~❤
```
```suggestion
あは、旗から出してあげたら、すっごく蕩けた顔してる❤\nわたしの匂いにつつまれながらしこしこ～ってしてもらえて、\nよかったわね。雑魚モンスターさん❤,Aha; let me pull you out from the flag. You have such a blissful expression on your face.❤ It was nice to be stroked while being wrapped in my scent right? You small fry monster~❤
```

Giratena replied:
```suggestion
あは、旗から出してあげたら、すっごく蕩けた顔してる❤\nわたしの匂いにつつまれながらしこしこ～ってしてもらえて、\nよかったわね。雑魚モンスターさん❤,Aha; I'll pull your penis out of the flag now. You have such a blissful expression on your face.❤ You enjoyed being stroked with something that smelled like me didn't you? You small fry monster~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+あは、旗から出してあげたら、すっごく蕩けた顔してる❤\nわたしの匂いにつつまれながらしこしこ～ってしてもらえて、\nよかったわね。雑魚モンスターさん❤,Aha; I'll pull you out from the flag; you are making such a melted face.❤ It was nice to be stroked while being wrapped in my scent right? You small fry monster~❤
+ううっ、我慢できない・・・❤,Auuuh; I can't hold back...❤
+んー？どうしたのって、あれ・・・。\nぷぷっ、わたしでオナニーし始めちゃったんだ❤,Mmh? What is it; ah... *Giggle*; so you started masturbating because of me.❤
+オナニーって、自分でおちんちんさわって射精しちゃうことでしょ？\nなんで男の子ってそんな風に自分から負けちゃうんだろうね。\nいまは大事な試合中なのになー❤,Masturbating is when you touch your penis by yourself to cum right? I wonder why boys make themselves lose on their own like that. Even though you are in the middle of an important battle~❤
```
```suggestion
オナニーって、自分でおちんちんさわって射精しちゃうことでしょ？\nなんで男の子ってそんな風に自分から負けちゃうんだろうね。\nいまは大事な試合中なのになー❤,Masturbation is the act of touching your own penis to make yourself cum; right? I wonder why boys would make themselves lose all on their own like that. Even though you're in the middle of an important battle~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ううっ、我慢できない・・・❤,Auuuh; I can't hold back...❤
+んー？どうしたのって、あれ・・・。\nぷぷっ、わたしでオナニーし始めちゃったんだ❤,Mmh? What is it; ah... *Giggle*; so you started masturbating because of me.❤
+オナニーって、自分でおちんちんさわって射精しちゃうことでしょ？\nなんで男の子ってそんな風に自分から負けちゃうんだろうね。\nいまは大事な試合中なのになー❤,Masturbating is when you touch your penis by yourself to cum right? I wonder why boys make themselves lose on their own like that. Even though you are in the middle of an important battle~❤
+ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; that's fine. If you are trying to lose for our sake; it means that big brother is your ally from now one!♪ I need to properly encourage you~❤
```
```suggestion
ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; that's fine. Losing can only benefit our side; which means that Mister is an ally of ours from now on!♪ I should properly encourage you~❤
```

Giratena replied:
```suggestion
ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; I don't care. If you lose; we win! Which means that you must be on our side now!♪ I should properly encourage you~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+んー？どうしたのって、あれ・・・。\nぷぷっ、わたしでオナニーし始めちゃったんだ❤,Mmh? What is it; ah... *Giggle*; so you started masturbating because of me.❤
+オナニーって、自分でおちんちんさわって射精しちゃうことでしょ？\nなんで男の子ってそんな風に自分から負けちゃうんだろうね。\nいまは大事な試合中なのになー❤,Masturbating is when you touch your penis by yourself to cum right? I wonder why boys make themselves lose on their own like that. Even though you are in the middle of an important battle~❤
+ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; that's fine. If you are trying to lose for our sake; it means that big brother is your ally from now one!♪ I need to properly encourage you~❤
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll get close to big brother's ear...❤Heeeey~ Hooooo~ Do your best stroking your penis...❤ Stroke; stroke; you are soooo cool...❤
```
```suggestion
ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll lean in close to Mister's ear...❤Heeeey~ Hooooo~ Do your best and stroke your penis...❤ Stroke; stroke; you're soooo cool...❤
```

Giratena replied:
```suggestion
ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Look; right by your ear I'll cheer…❤ Go-go stroking; work that cock without fear…❤ Stroke-stroke; stroke-stroke…❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+オナニーって、自分でおちんちんさわって射精しちゃうことでしょ？\nなんで男の子ってそんな風に自分から負けちゃうんだろうね。\nいまは大事な試合中なのになー❤,Masturbating is when you touch your penis by yourself to cum right? I wonder why boys make themselves lose on their own like that. Even though you are in the middle of an important battle~❤
+ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; that's fine. If you are trying to lose for our sake; it means that big brother is your ally from now one!♪ I need to properly encourage you~❤
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll get close to big brother's ear...❤Heeeey~ Hooooo~ Do your best stroking your penis...❤ Stroke; stroke; you are soooo cool...❤
+はう、はううっ❤ぞくぞくするぅ・・❤,Ha; hauu; it's making me shiver...❤
```
```suggestion
はう、はううっ❤ぞくぞくするぅ・・❤,Ha; hauu; I feel all tingly...❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ま、いいや。わたしたちのために負けてくれるっていうなら、\nお兄さんは今から味方♪ちゃんと応援してあげないとね❤,Well; that's fine. If you are trying to lose for our sake; it means that big brother is your ally from now one!♪ I need to properly encourage you~❤
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll get close to big brother's ear...❤Heeeey~ Hooooo~ Do your best stroking your penis...❤ Stroke; stroke; you are soooo cool...❤
+はう、はううっ❤ぞくぞくするぅ・・❤,Ha; hauu; it's making me shiver...❤
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、さっきより一生懸命にしこしこできて、\nえらいぞー❤,*Giggle*❤ You are so good at stroking~❤ Compared to earlier; my cheers made you stroke so eagerly; good job~❤
```
```suggestion
くすくす❤しこしこするの上手だね❤\n応援にこたえて、さっきより一生懸命にしこしこできて、\nえらいぞー❤,*Giggle*❤ You're so good at stroking~❤ Compared to earlier; you're stroking so much more eagerly thanks to my cheering; good job~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちんしこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll get close to big brother's ear...❤Heeeey~ Hooooo~ Do your best stroking your penis...❤ Stroke; stroke; you are soooo cool...❤
+はう、はううっ❤ぞくぞくするぅ・・❤,Ha; hauu; it's making me shiver...❤
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、さっきより一生懸命にしこしこできて、\nえらいぞー❤,*Giggle*❤ You are so good at stroking~❤ Compared to earlier; my cheers made you stroke so eagerly; good job~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your fans; stroke; stroke; stroke; stroke~♪ Until you cum; let's keep going without stopping your hand at all~❤
```
```suggestion
おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Do your best; Mister Penis~❤... Do your best; Mister~❤... We may be in the midst of battle; but it's okay to betray your fans' expectations. Stroke; stroke; stroke; stroke~♪ Don't stop stroking that hand until you cum~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+はう、はううっ❤ぞくぞくするぅ・・❤,Ha; hauu; it's making me shiver...❤
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、さっきより一生懸命にしこしこできて、\nえらいぞー❤,*Giggle*❤ You are so good at stroking~❤ Compared to earlier; my cheers made you stroke so eagerly; good job~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your fans; stroke; stroke; stroke; stroke~♪ Until you cum; let's keep going without stopping your hand at all~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出しちゃえー・・・❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your allies; stroke stroke; stroke stroke~❤ Feel good and pour out plenty of energy for us~...❤
```
```suggestion
おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出しちゃえー・・・❤,Do your best; Mister Penis~❤... Do your best; Mister~❤... We may be in the midst of battle; but go ahead and betray your allies. Stroke; stroke; stroke; stroke~♪ Just let yourself feel good and pour out lots of energy for me...❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、さっきより一生懸命にしこしこできて、\nえらいぞー❤,*Giggle*❤ You are so good at stroking~❤ Compared to earlier; my cheers made you stroke so eagerly; good job~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your fans; stroke; stroke; stroke; stroke~♪ Until you cum; let's keep going without stopping your hand at all~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出しちゃえー・・・❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your allies; stroke stroke; stroke stroke~❤ Feel good and pour out plenty of energy for us~...❤
+わたしに応援されるとどんどん元気になっちゃうね❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃうね❤\n自分で触ってるだけで、あへあへになっちゃうよわーいおちんちん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Getting all flustered from touching your weaaaaak penis.❤
```
```suggestion
わたしに応援されるとどんどん元気になっちゃうね❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃうね❤\n自分で触ってるだけで、あへあへになっちゃうよわーいおちんちん❤,Seems like you're getting more and more excited from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Getting all flustered from touching your weaaaaak penis.❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your fans; stroke; stroke; stroke; stroke~♪ Until you cum; let's keep going without stopping your hand at all~❤
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出しちゃえー・・・❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your allies; stroke stroke; stroke stroke~❤ Feel good and pour out plenty of energy for us~...❤
+わたしに応援されるとどんどん元気になっちゃうね❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃうね❤\n自分で触ってるだけで、あへあへになっちゃうよわーいおちんちん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Getting all flustered from touching your weaaaaak penis.❤
+ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; it's fine if you don't hold it in~❤ Big brother's perseverance is impressive.❤ You're so cool.❤ So it's fine to cum blissfully~❤
```
```suggestion
ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; there's no need to hold back~❤ Mister is already so cool;❤ holding out so long;❤ so it's fine if you happily let it out now.❤
```

Giratena replied:
```suggestion
ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; there's no need to hold back~❤ Everyone already know's how cool you are~❤ You've been holding out for so long already; so OK if you want to let everything out now~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+おちんちんさんがんばれー・・・❤おにいさんもがんばれー・・・❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出しちゃえー・・・❤,Mr. Penis; do your best~❤... Big brother; do your best~❤... We are in the middle of a battle; but betray your allies; stroke stroke; stroke stroke~❤ Feel good and pour out plenty of energy for us~...❤
+わたしに応援されるとどんどん元気になっちゃうね❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃうね❤\n自分で触ってるだけで、あへあへになっちゃうよわーいおちんちん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Getting all flustered from touching your weaaaaak penis.❤
+ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; it's fine if you don't hold it in~❤ Big brother's perseverance is impressive.❤ You're so cool.❤ So it's fine to cum blissfully~❤
+出して、出して❤わたしの応援に合わせて、\n手を早く動かして、びゅーって気持ちよくおもらしして❤,Cum; cum !❤ Stroke it fast in rhythm with my cheers; and reach a pleasant orgasm~❤
```
```suggestion
出して、出して❤わたしの応援に合わせて、\n手を早く動かして、びゅーって気持ちよくおもらしして❤,Let it out; just let it all out.❤ Stroke it quickly in rhythm with my cheers; until you achieve a pleasant orgasm~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+わたしに応援されるとどんどん元気になっちゃうね❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃうね❤\n自分で触ってるだけで、あへあへになっちゃうよわーいおちんちん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Getting all flustered from touching your weaaaaak penis.❤
+ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; it's fine if you don't hold it in~❤ Big brother's perseverance is impressive.❤ You're so cool.❤ So it's fine to cum blissfully~❤
+出して、出して❤わたしの応援に合わせて、\n手を早く動かして、びゅーって気持ちよくおもらしして❤,Cum; cum !❤ Stroke it fast in rhythm with my cheers; and reach a pleasant orgasm~❤
+ほら、さん、はい♪\nふれー❤ふれー❤おにいさん♪\nしこしこ♪しこしこ♪びゅっ、びゅっ、びゅーー❤,And... start! Here we go~♪ Keep going~❤ keep going~❤ Big brother!♪ Stroke stroke;♪ stroke stroke;♪ spurt; spurt; spuuuuuurt~❤
```
```suggestion
ほら、さん、はい♪\nふれー❤ふれー❤おにいさん♪\nしこしこ♪しこしこ♪びゅっ、びゅっ、びゅーー❤,And... start! Here we go~♪ Keep going~❤ keep going~❤ Mister!♪ Stroke stroke;♪ stroke stroke;♪ spurt; spurt; spuuuuuurt~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ほら、がまんなんてしなくていいんだよー❤\nおにいさんはがんばっててすごい❤とってもかっこいい❤\nだから気持ちよく、びゅってしちゃっていいの❤,Come on; it's fine if you don't hold it in~❤ Big brother's perseverance is impressive.❤ You're so cool.❤ So it's fine to cum blissfully~❤
+出して、出して❤わたしの応援に合わせて、\n手を早く動かして、びゅーって気持ちよくおもらしして❤,Cum; cum !❤ Stroke it fast in rhythm with my cheers; and reach a pleasant orgasm~❤
+ほら、さん、はい♪\nふれー❤ふれー❤おにいさん♪\nしこしこ♪しこしこ♪びゅっ、びゅっ、びゅーー❤,And... start! Here we go~♪ Keep going~❤ keep going~❤ Big brother!♪ Stroke stroke;♪ stroke stroke;♪ spurt; spurt; spuuuuuurt~❤
+あはは❤おもいっきり射精できてえらいねー❤\n見てみて、こんなに出ちゃってる❤\nお兄さんのエナジーがいっぱいもらえて、んー、おいしい♪,Ahaha~❤You came so much; good job~❤ Look; look; I got a lot of Big Brother's energy since you came so much; mmmh; yummy!♪
```
```suggestion
あはは❤おもいっきり射精できてえらいねー❤\n見てみて、こんなに出ちゃってる❤\nお兄さんのエナジーがいっぱいもらえて、んー、おいしい♪,Ahaha~❤You came so much; good job~❤ Look; look; I got plenty of of Mister's energy since you came so much; mmmh; yummy!♪
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+出して、出して❤わたしの応援に合わせて、\n手を早く動かして、びゅーって気持ちよくおもらしして❤,Cum; cum !❤ Stroke it fast in rhythm with my cheers; and reach a pleasant orgasm~❤
+ほら、さん、はい♪\nふれー❤ふれー❤おにいさん♪\nしこしこ♪しこしこ♪びゅっ、びゅっ、びゅーー❤,And... start! Here we go~♪ Keep going~❤ keep going~❤ Big brother!♪ Stroke stroke;♪ stroke stroke;♪ spurt; spurt; spuuuuuurt~❤
+あはは❤おもいっきり射精できてえらいねー❤\n見てみて、こんなに出ちゃってる❤\nお兄さんのエナジーがいっぱいもらえて、んー、おいしい♪,Ahaha~❤You came so much; good job~❤ Look; look; I got a lot of Big Brother's energy since you came so much; mmmh; yummy!♪
+わたしけっこうダメージ受けてたからお兄さんのサポート、\n助かっちゃった♪\nわたしたちのためにシコシコしてくれてありがとうねー❤,I took quite a lot of damage; so Big Brother's help is a big help.♪ Thank you for stroking it for our sake~❤
```
```suggestion
わたしけっこうダメージ受けてたからお兄さんのサポート、\n助かっちゃった♪\nわたしたちのためにシコシコしてくれてありがとうねー❤,I took quite a lot of damage; so Mister's contribution is a big help.♪ Thank you for stroking it for our sake~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+ほら、さん、はい♪\nふれー❤ふれー❤おにいさん♪\nしこしこ♪しこしこ♪びゅっ、びゅっ、びゅーー❤,And... start! Here we go~♪ Keep going~❤ keep going~❤ Big brother!♪ Stroke stroke;♪ stroke stroke;♪ spurt; spurt; spuuuuuurt~❤
+あはは❤おもいっきり射精できてえらいねー❤\n見てみて、こんなに出ちゃってる❤\nお兄さんのエナジーがいっぱいもらえて、んー、おいしい♪,Ahaha~❤You came so much; good job~❤ Look; look; I got a lot of Big Brother's energy since you came so much; mmmh; yummy!♪
+わたしけっこうダメージ受けてたからお兄さんのサポート、\n助かっちゃった♪\nわたしたちのためにシコシコしてくれてありがとうねー❤,I took quite a lot of damage; so Big Brother's help is a big help.♪ Thank you for stroking it for our sake~❤
+全然ダメージ受けてなかったけど、\nでもエナジーくれてうれしいよ❤\nわたしたちのためにシコシコしてくれてありがとうねー❤,I didn't take any damage; but I'm happy you still gave me your energy~❤ Thank you for stroking it for our sake~❤
```
```suggestion
全然ダメージ受けてなかったけど、\nでもエナジーくれてうれしいよ❤\nわたしたちのためにシコシコしてくれてありがとうねー❤,I didn't take any damage; but I'm happy you gave me your energy regardless~❤ Thank you for stroking it for our sake~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+あはは❤おもいっきり射精できてえらいねー❤\n見てみて、こんなに出ちゃってる❤\nお兄さんのエナジーがいっぱいもらえて、んー、おいしい♪,Ahaha~❤You came so much; good job~❤ Look; look; I got a lot of Big Brother's energy since you came so much; mmmh; yummy!♪
+わたしけっこうダメージ受けてたからお兄さんのサポート、\n助かっちゃった♪\nわたしたちのためにシコシコしてくれてありがとうねー❤,I took quite a lot of damage; so Big Brother's help is a big help.♪ Thank you for stroking it for our sake~❤
+全然ダメージ受けてなかったけど、\nでもエナジーくれてうれしいよ❤\nわたしたちのためにシコシコしてくれてありがとうねー❤,I didn't take any damage; but I'm happy you still gave me your energy~❤ Thank you for stroking it for our sake~❤
+んふふ❤お兄さんもオナニー？\nわたしたちのために、エナジーぴゅっぴゅっしてくれるんだ❤\nやさしーい❤,*Giggles*❤ You are masturbating too Big Brother ? You'll spurt out your energy for us; you're so kind~❤
```
```suggestion
んふふ❤お兄さんもオナニー？\nわたしたちのために、エナジーぴゅっぴゅっしてくれるんだ❤\nやさしーい❤,*Giggles*❤ You started masturbating too; Mister? Spurting out your energy for us; you're so kind~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+わたしけっこうダメージ受けてたからお兄さんのサポート、\n助かっちゃった♪\nわたしたちのためにシコシコしてくれてありがとうねー❤,I took quite a lot of damage; so Big Brother's help is a big help.♪ Thank you for stroking it for our sake~❤
+全然ダメージ受けてなかったけど、\nでもエナジーくれてうれしいよ❤\nわたしたちのためにシコシコしてくれてありがとうねー❤,I didn't take any damage; but I'm happy you still gave me your energy~❤ Thank you for stroking it for our sake~❤
+んふふ❤お兄さんもオナニー？\nわたしたちのために、エナジーぴゅっぴゅっしてくれるんだ❤\nやさしーい❤,*Giggles*❤ You are masturbating too Big Brother ? You'll spurt out your energy for us; you're so kind~❤
+わたしたちのために負けてくれるっていうなら、\nお兄さんも今から味方だよ。\nちゃんと応援してあげないとね❤,Since you are trying to lose for us; that means Big Brother is our ally now. I need to properly encourage you.❤
```
```suggestion
わたしたちのために負けてくれるっていうなら、\nお兄さんも今から味方だよ。\nちゃんと応援してあげないとね❤,Since you are trying to lose for us; that means you're our ally now; Mister. I ought to properly encourage you.❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+全然ダメージ受けてなかったけど、\nでもエナジーくれてうれしいよ❤\nわたしたちのためにシコシコしてくれてありがとうねー❤,I didn't take any damage; but I'm happy you still gave me your energy~❤ Thank you for stroking it for our sake~❤
+んふふ❤お兄さんもオナニー？\nわたしたちのために、エナジーぴゅっぴゅっしてくれるんだ❤\nやさしーい❤,*Giggles*❤ You are masturbating too Big Brother ? You'll spurt out your energy for us; you're so kind~❤
+わたしたちのために負けてくれるっていうなら、\nお兄さんも今から味方だよ。\nちゃんと応援してあげないとね❤,Since you are trying to lose for us; that means Big Brother is our ally now. I need to properly encourage you.❤
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちん♪しこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,I'll get close to your ear and...❤ do! your! best! stroke your penis♪ do your best...❤ Stroke; stroke; you're so cool...❤
```
```suggestion
ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちん♪しこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,I'll lean in close to your ear and...❤ Do~! your~! best~! Stroke that penis♪ You can do it...❤ Stroke; stroke; you're so cool...❤
```

Giratena replied:
```suggestion
ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちん♪しこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,Here; I'll whisper in your ear…❤ Go-go cock! Keep stroking; have no fear…❤ Stroke-stroke; be my guest~ Pump-pump; you're the best!❤
```

Giratena commented:
Hey @WisDZ!

Replied to the comments you left a couple of hours ago with some suggestions! Hopefully you like them. Also I see @S-h-r-u-g  left some stuff too! Looking good!

I'll also mark this PR as ready for review now since everything is done now and is ready to be looked at ^^

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+わたしたちのために負けてくれるっていうなら、\nお兄さんも今から味方だよ。\nちゃんと応援してあげないとね❤,Since you are trying to lose for us; that means Big Brother is our ally now. I need to properly encourage you.❤
+ほら、お兄さんの耳の近くで・・・❤\nふれーふれー、おちんちん♪しこしこがんばれー・・・❤\nしこしこ、しこしこ、かっこいいぞー・・・❤,I'll get close to your ear and...❤ do! your! best! stroke your penis♪ do your best...❤ Stroke; stroke; you're so cool...❤
+はう、はううっ・・❤,Ha-a; Hau-u-u...❤
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、一生懸命にしこしこできて、\nえらいぞー❤,*Giggles*❤ You're so skilled at stroking your penis~❤ You are stroking your penis with all your might thanks to my cheers; gooooood job~❤
```
```suggestion
くすくす❤しこしこするの上手だね❤\n応援にこたえて、一生懸命にしこしこできて、\nえらいぞー❤,*Giggles*❤ You're so skilled at stroking your penis~❤ You're stroking your penis so vigorously thanks to my cheers; gooooood job~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+はう、はううっ・・❤,Ha-a; Hau-u-u...❤
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、一生懸命にしこしこできて、\nえらいぞー❤,*Giggles*❤ You're so skilled at stroking your penis~❤ You are stroking your penis with all your might thanks to my cheers; gooooood job~❤
+仲間が応援されてるの見てたら、ぼくも応援されたいよーって\n我慢できなくなっちゃったんだよね❤\nわたしのかわいい声で、がんばれ～って言ってほしかったんだよね❤,When you saw your ally getting cheered on by me; you thought "I-I want to be cheered on too"; right? You couldn't hold back; right?❤ You wanted me to encourage you with my cute voice; right...❤
+ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr Penis; do your best~❤ Big Brother; do your best~❤ We are in the middle of a battle; but betray all your fans and stroke yourself a lot.♪ Let's keep going until you spurt~❤
```
```suggestion
ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mister Penis; do your best~❤ You too; Mister; do your best~❤ We're in the middle of a battle; but go ahead and stroke a bunch; betraying all your fans.♪ Let's keep going until you spurt~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+くすくす❤しこしこするの上手だね❤\n応援にこたえて、一生懸命にしこしこできて、\nえらいぞー❤,*Giggles*❤ You're so skilled at stroking your penis~❤ You are stroking your penis with all your might thanks to my cheers; gooooood job~❤
+仲間が応援されてるの見てたら、ぼくも応援されたいよーって\n我慢できなくなっちゃったんだよね❤\nわたしのかわいい声で、がんばれ～って言ってほしかったんだよね❤,When you saw your ally getting cheered on by me; you thought "I-I want to be cheered on too"; right? You couldn't hold back; right?❤ You wanted me to encourage you with my cute voice; right...❤
+ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr Penis; do your best~❤ Big Brother; do your best~❤ We are in the middle of a battle; but betray all your fans and stroke yourself a lot.♪ Let's keep going until you spurt~❤
+ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出してー❤,Mr Penis; do your best~❤ Big Brother; do your best~❤ We are in the middle of a battle; but betray your allies; stroke; stroke; stroke.❤ Feel good and pour out plenty of energy for us~❤
```
```suggestion
ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出してー❤,Mister Penis; do your best~❤ You too; Mister; do your best~❤ We're in the middle of a battle; but feel free to betray your allies; stroke; stroke; stroke.❤ Feel good and pour out plenty of energy for us~❤
```

S-h-r-u-g commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+仲間が応援されてるの見てたら、ぼくも応援されたいよーって\n我慢できなくなっちゃったんだよね❤\nわたしのかわいい声で、がんばれ～って言ってほしかったんだよね❤,When you saw your ally getting cheered on by me; you thought "I-I want to be cheered on too"; right? You couldn't hold back; right?❤ You wanted me to encourage you with my cute voice; right...❤
+ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、ファンのみんなのこと裏切って、\nしこしこしこしこ♪手を止めずにびゅびゅっしようねー❤,Mr Penis; do your best~❤ Big Brother; do your best~❤ We are in the middle of a battle; but betray all your fans and stroke yourself a lot.♪ Let's keep going until you spurt~❤
+ほら、おちんちんさんがんばれー…❤おにいさんがんばれー…❤\n試合中だけど、仲間のこと裏切って、しこしこ、しこしこ❤\n気持ちよくなってわたしたちのためにエナジー出してー❤,Mr Penis; do your best~❤ Big Brother; do your best~❤ We are in the middle of a battle; but betray your allies; stroke; stroke; stroke.❤ Feel good and pour out plenty of energy for us~❤
+わたしに応援されるとどんどん元気になっちゃう❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃう❤\n自分で触ってるだけなのに、頭の中があへあへぇ❤お顔もとろーん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ Your just touching yourself; but your head is becoming stupid; and your face is melting~❤
```
```suggestion
わたしに応援されるとどんどん元気になっちゃう❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃう❤\n自分で触ってるだけなのに、頭の中があへあへぇ❤お顔もとろーん❤,Seems like you're getting in the mood more and more from my cheering~❤ Your penis is becoming harder; and you're getting more sensitive too~❤ You're just touching yourself; but your head is becoming stupid; and your face is melting~❤
```

Giratena replied:
```suggestion
わたしに応援されるとどんどん元気になっちゃう❤\nおちんちんどんどん固くなって、もっともっと敏感になっちゃう❤\n自分で触ってるだけなのに、頭の中があへあへぇ❤お顔もとろーん❤,Seems like my cheering is really getting your cock worked up~❤ I can see it getting harder; and I bet you're more sensitive too~❤ You're just touching yourself; but you're already getting dumber and making a silly face~❤
```

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: Sayshi <22719296+Theguysayhi@users.noreply.github.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: Sayshi <22719296+Theguysayhi@users.noreply.github.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Update CSV/Language/English/Card/10067.txt

Giratena committed:
Apply suggestion from @Giratena

Giratena commented on CSV/Language/English/Card/10067.txt:
```diff
@@ -1,345 +1,345 @@
+寝転んだお兄さんの股に足を当ててー、\nたまたまから先っぽまで、足でごしごし♪ごしごし❤\n精液さんでてこーい❤って、こすってあげるの❤,Since you're just laying on the floor; I'll press my foot on your crotch and... from your balls to the tip of your penis; I'll grind; grind~♪ stroke; stroke~♪ with my legs~❤ I'll stroke you and coax your semen to~ come~❤ out~❤
+何回こすったところでおもらししちゃうかなー❤\n数えててあげるね。ほら、\nいーち♪、にー♪くすくす❤,How many strokes will it take for you to cum; I wonder~❤ Let's see. Here's oooooooone~♪ twooooooo~♪ *Giggle*
+まわりがみんな爆乳だから、かすんじゃうけど、\nわたしだって意外とおっぱいあるんだよ？\nがんばれば、おちんちんだって挟めるんだから、えいっ❤むにむに～❤,Everyone here has enormous breasts; so I know I can't compare with them... But you have to admit; my boobs are still pretty big; right? If I try; I can sandwich your penis between them; here I come! Hah!❤
+すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right.❤ I'll rub you plenty; so don't try to pull away~❤ I'll take one breast; and the other; and... rub; rub!❤ again and again!❤
```
```suggestion
すべすべおっぱい、気持ちいいでしょ❤\nいっぱいズリズリしてあげるから、逃げちゃダメだよー❤\nよいしょっ、よいしょっ、ずりずり❤ずりずり❤,My smooth breasts feel good right?❤ I'll rub it plenty; so don't try to pull away~❤ I'll just cup them in my hands and... rub them togehter like this~!❤ And this~!❤
```

Giratena committed:
Apply suggestion from @S-h-r-u-g

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Apply suggestion from @S-h-r-u-g

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena committed:
Apply suggestions from code review

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>

Giratena reviewed (commented):
Alterations from me

Giratena committed:
Apply suggestions from code review

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>
Co-authored-by: GiratenaTranslations <57820839+Giratena@users.noreply.github.com>

Giratena committed:
Apply suggestions from code review

Co-authored-by: S-h-r-u-g <jlmtruth602@gmail.com>
Co-authored-by: GiratenaTranslations <57820839+Giratena@users.noreply.github.com>

S-h-r-u-g committed:
Update 10067.txt

Giratena committed:
First wave of proofreading

Giratena committed:
Cleaned up format
